spaces/templates/post_edit.html

```diff
@@ -116,4 +116,122 @@ <h2 >Box Your Post</h2>
 
 
 
+{% endblock %}{% extends 'base.html' %}
```

Comment:
@MariaAsif135 @cemrekut This part should be changed. Please delete this part from the dev.